### PR TITLE
Refs #25232 -- Added tests for AllowInactiveUsersModelBackend authentication backend

### PR DESCRIPTION
**Pull request by sasha0**

### CODE REVIEW

File: tests/auth_tests/test_auth_backends.py [hunk 1]
```diff
@@ -667,3 +667,48 @@ def test_backend_path_login_with_explicit_backends(self):
...
+    """
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = User.objects.create_user(
+            username='test',
+            email='test@example.com',
+            password='test'
+        )
+        cls.user.is_active = False
+        cls.user.save()
```
Comment (charettes) [inline tests/auth_tests/test_auth_backends.py]: You should be able to pass `is_active=False` to `create_user()`.

File: tests/auth_tests/test_auth_backends.py [hunk 2]
```diff
@@ -667,3 +667,48 @@ def test_backend_path_login_with_explicit_backends(self):
...
+        self.assertEqual(
+            authenticate(
+                username='test',
+                email='test@example.com',
+                password='test'
+            ),
+            self.user
+        )
+
+    def test_get_user(self):
```
Comment (timgraham) [inline tests/auth_tests/test_auth_backends.py]: What I meant was keep this test in the original PR (it passes there, right?) and move the other one that uses `login()` to https://github.com/django/django/pull/6124.

Comment (sasha0) [inline tests/auth_tests/test_auth_backends.py]: No, these whole test case won't pass there - authentication backend `django.contrib.auth.backends.AllowInactiveUsersModelBackend` exists only in #6090.

Comment (timgraham) [inline tests/auth_tests/test_auth_backends.py]: `test_authenticate` can be part of #6090, we'll only merge #6124 after #6090 is merged so `test_get_user` could be part of that patch. Actually could we change `test_get_user` to use `self.client.force_login(self.user)` and include both tests in #6090?

Comment (sasha0) [inline tests/auth_tests/test_auth_backends.py]: Yep, returned both tests to #6090 and going to close this PR.

**Pull request closed without merging**